RackAndPower/PowerShelfManagementSpec.md

```diff
@@ -0,0 +1,644 @@
+        }
+    }
+}
+```
+
+## Get branch circuits
+
+The `Circuit` resources in the `Branches` collection represents the outputs from the power shelf with associated readings.
+
+```
+GET /redfish/v1/PowerEquipment/PowerShelves/1/Branches/DC
+
+{
+    "@odata.id": "/redfish/v1/PowerEquipment/PowerShelves/1/Branches/DC",
+    "@odata.type": "#Circuit.v1_7_0.Circuit",
+    "Id": "DC",
+    "Name": "DC Output Circuit to Busbar",
+    "Status": {
+        "State": "Enabled",
+        "Health": "OK"
+    },
+    "CircuitType": "Bus",
+    "VoltageType": "DC",
+    "NominalVoltage": "DC48V",
+    "RatedCurrentAmps": 250,
+    "BreakerState": "Normal",
+    "Voltage": {
+        "DataSourceUri": "/redfish/v1/Chassis/PowerShelf/Sensors/VoltageDC",
+        "Reading": 48.45
+    },
+    "CurrentAmps": {
+        "DataSourceUri": "/redfish/v1/Chassis/PowerShelf/Sensors/CurrentDC",
+        "Reading": 16.93
+    },
+    "PowerWatts": {
+        "DataSourceUri": "/redfish/v1/Chassis/PowerShelf/Sensors/PowerDC",
+        "Reading": 816.5
+    },
+    "EnergykWh": {
+        "DataSourceUri": "/redfish/v1/Chassis/PowerShelf/Sensors/EnergyDC",
+        "Reading": 121666
+    },
+    "Links": {
+        "DistributionCircuits": [
+            {
+                "@odata.id": "/redfish/v1/PowerEquipment/ElectricalBuses/Rack42Busbar/Mains/DC"
+            }
+        ]
+    }
+}
+```
+
+## Get outlets
+
+The `Outlet` resources in the `Outlets` collection represents recepticals directly connected to equipment from the power shelf with associated readings.
+
+```
+GET /redfish/v1/PowerEquipment/PowerShelves/1/Branches/DC
+
+{
+    "@odata.id": "/redfish/v1/PowerEquipment/PowerShelves/1/Outlets/A1",
+    "@odata.type": "#Outlet.v1_4_1.Outlet",
+    "Id": "A1",
+    "Name": "Outlet A1",
+    "Status": {
+        "Health": "OK",
+        "State": "Enabled"
+    },
+    "PhaseWiringType": "OnePhase3Wire",
+    "VoltageType": "AC",
+    "OutletType": "NEMA_5_15R",
+    "RatedCurrentAmps": 15,
+    "NominalVoltage": "AC120V",
+    "LocationIndicatorActive": true,
+    "PowerOnDelaySeconds": 4,
+    "PowerOffDelaySeconds": 0,
+    "PowerState": "On",
+    "PowerEnabled": true,
+    "Voltage": {
+        "DataSourceUri": "/redfish/v1/Chassis/PowerShelf/Sensors/VoltageA",
+        "Reading": 121.4
+    },
+    "CurrentAmps": {
+        "DataSourceUri": "/redfish/v1/Chassis/PowerShelf/Sensors/CurrentA",
+        "Reading": 1.59
+    },
+    "PowerWatts": {
+        "DataSourceUri": "/redfish/v1/Chassis/PowerShelf/Sensors/PowerA",
+        "Reading": 192.4
+    },
+    "Links": {
+        "BranchCircuit": {
+            "@odata.id": "/redfish/v1/PowerEquipment/PowerShelves/1/Branches/A"
+        }
+    }
+}
+```
+
+# References
+
+\[1\] Usage Guide and Requirements for the OCP Baseline Hardware Management Profile v1.0.1
+\[2\] "Redfish API Specification" - [*https://www.dmtf.org/dsp/DSP0266*](https://www.dmtf.org/dsp/DSP0266)
+

```

Comment:
Add reference to Redfish Data Model Specification and Redfish Interoperability Profiles Specification

Reply:
Done